src/EFCore.PG/Storage/Internal/NpgsqlDatabaseCreator.cs

```diff
@@ -317,9 +317,13 @@ public override void CreateTables()
             var commands = Dependencies.MigrationsSqlGenerator.Generate(operations, Dependencies.Model);
 
             // If a PostgreSQL extension or enum was added, we want Npgsql to reload all types at the ADO.NET level.
```

Comment:
Not only an extension or enum, but also a range.

Reply:
Done